server/http_test.go

```diff
@@ -332,6 +332,125 @@ func TestHandleAuthFuncResponsesMultipleRedirectURLs(t *testing.T) {
 	}
 }
 
+func TestHandleTokenFunc(t *testing.T) {
+
+	fx, err := makeTestFixtures()
+	if err != nil {
+		t.Errorf("could not run test fixtures")
```

Comment:
Need to `t.Fatal` here since the test can't continue if this fails. Also please print the error:

```
t.Fatalf("could not run test fixtures: %v", err)
```